addons/common/fnc_filterLoadout.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_filterLoadout
+Description:
+    Used to filter a getUnitLoadout array.
+Parameters:
+    _loadout - getUnitLoadout array <ARRAY>
+Example:
+    (begin example)
+        _loadout = [getUnitLoadout player] call CBA_fnc_filterLoadout;
+    (end)
+Returns:
+    filtered getUnitLoadout array <ARRAY>
+Author:
+    SynixeBrett
+---------------------------------------------------------------------------- */
+SCRIPT(filterLoadout);
+
+params [["_loadout", [], [[]]]];
```

Comment:
Should handle receiving the different inputs (OBJECT, STRING, CONFIG) for the `getUnitLoadout` command as well as a normal loadout array.